### PR TITLE
[14.0][MIG] account_bank_statement_import_move_line > account_statement_import_move_line

### PR DESCRIPTION
**Pull request by LudLaf**
From v13 to v14 module has been renamed like so:
(account_bank_statement_import_move_line > account_statement_import_move_line).

@Tecnativa TT33537

### CODE REVIEW

**Comment by LudLaf** [comment]
@carlosdauden @pedrobaeza please review

File: account_statement_import_move_line/tests/test_account_bank_statement_import_move_line.py [hunk 1]
```diff
@@ -22,17 +22,33 @@ def setUpClass(cls):
...
         cls.journal = cls.env["account.journal"].create(
             {"name": "Test Journal", "type": "sale", "code": "TJS0"}
         )
+        cls.bank_journal = cls.env["account.journal"].create(
+            {
+                "name": "Bank Journal - (test)",
+                "code": "TBNK",
+                "type": "bank",
+                "suspense_account_id": cls.a_suspense.id,
+                # "currency_id": cls.env.ref("base.USD").id,
```
Comment (carlosdauden) [inline account_statement_import_move_line/tests/test_account_bank_statement_import_move_line.py]: Remove commented line

**Review by carlosdauden — APPROVED** [review]

File: account_statement_import_move_line/tests/test_account_bank_statement_import_move_line.py (hunk 1 above)
Comment (LudLaf) [reply account_statement_import_move_line/tests/test_account_bank_statement_import_move_line.py]: Requested changes now done.

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**